components/service/glean/src/test/java/mozilla/components/service/glean/GleanTest.kt

```diff
@@ -115,13 +119,20 @@ class GleanTest {
             )
             Glean.setExperimentActive(
                 "experiment2", "branch_b",
-                    mapOf("key" to "value")
+                mapOf("key" to "value")
             )
             Glean.setExperimentInactive("experiment1")
 
+            Glean.metricsPingScheduler.clearSchedulerData()
             Glean.handleEvent(Glean.PingEvent.Default)
 
-            val request = server.takeRequest()
+            // Check that the work is scheduled
+            assertTrue(isWorkScheduled())
```

Comment:
I think we always want this assert whenever we call `triggerWorkManager`: maybe we can move it within that function?